.do/k8s-cluster/Makefile

```diff
@@ -59,6 +59,30 @@ install_nginx:
 	helm upgrade --install nginx-ingress --version ${NGINX_INGRESS_VERSION} --namespace kube-system ingress-nginx/ingress-nginx \
 	--set controller.publishService.enabled=true
 
+.PHONY: install_certmanager
```

Comment:
This will only run when merges occur against master. That's the only time where we provision infrastructure.